### PR TITLE
Action button for treatment legend

### PR DESCRIPTION
**Pull request by lastminutediorama**
Adds a button at the top right of the map that display the Treatment Legend if we're in "treatment layers mode" and the legend was closed. Clicking the button re-opens the legend.

### CODE REVIEW

File: src/interface/src/app/treatments/treatment-map/treatment-map.component.html [hunk 1]
```diff
@@ -23,6 +23,15 @@
     *ngIf="showMapControls$ | async"
     [mapLibreMap]="mapLibreMap"></app-map-controls>
 
+  <app-action-button
+    *ngIf="
+      (standSelectionEnabled$ | async) === false &&
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/treatment-map/treatment-map.component.html:28]: I think we should have the button and legend disappear in stand editing mode, and I think this is the only var that designates that mode. Happy to change this, if there's a better approach.

File: src/interface/src/app/treatments/map-actionbutton/map-actionbutton.component.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<mgl-control position="top-right">
```
Comment (pflopez) [inline src/interface/src/app/treatments/map-actionbutton/map-actionbutton.component.html:1]: Can you rename these files to map-action-button ? Seems like we are missing a `-`

File: src/interface/src/app/treatments/map-actionbutton/map-actionbutton.component.spec.ts
Comment (pflopez) [inline src/interface/src/app/treatments/map-actionbutton/map-actionbutton.component.spec.ts:1]: Why leave this empty file?

**Review by pflopez — COMMENTED** [review]
Looking good! - Just found a couple of details. 
Not sure if all the TODOs on the description are still pending.

**Review by pflopez — APPROVED** [review]

**Pull request merged**